Automattic/sensei
# Add WCCOM connect notice

Fixes #4151

### Changes proposed in this Pull Request

* Adds a notice on the courses page, plugins page, learners page, and extensions page when WooCommerce isn't connected to WooCommerce.com but WC Sensei extensions are installed.
* Updates the SVG used on the block category.
  * Since we need the SVG for non-React pages, I couldn't figure out how to import the `svg` file into our SenseiIcon component with the props. If anyone has any ideas, let me know. It'd be nice to not duplicate that.
* Hacked Calypso Build to fix our assets in the `FileLoader` plugin. 
* It currently has to go through flow and return to WC extensions page. We could possibly hack it with a transient, but it would be a bit messy. I can try to do this if we think necessary.

### Testing instructions

* Ensure this doesn't show up when no Sensei WC.com extensions are installed.
* Ensure it does show up when WC.com extensions are installed.
